demo/content/View/index.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react'
+import {
+   Grid,
+   GridCell,
+   Spacer,
+   Text,
+   Title,
+   TitleBar,
+   View
+} from '../../../src'
+
+import style from '../../style.less'
+
+export default class DemoContentView extends Component {
+   static propTypes = {
+      contents: React.PropTypes.array
+   }
+
+   render() {
+      return (
+         <View width="100%" className={style.docs}>
+            <TitleBar title="View" flush />
+            <Spacer size="9" />
+            <Grid columns="1[a] 2[b] 4[c-e]" gutter="60px">
+               <GridCell>
+                  <Title>Introduction</Title>
+               </GridCell>
+               <GridCell colspan="3">
+                  <Text size="6" weight="light">
+                     The View component is probably one of the most important components within OIO.
+                     It is the core OIO component you should use to structure your pages, layouts and components for your application.
+                     You can think of the <code>View</code> component as OIO‘s take on a <code>div</code>.
```

Comment:
hmm yeah I noticed that. I think that's what happens when I use that 'safe' apostrophe (Option-])